clusterloader2/pkg/prometheus/manifests/exporters/kube-state-metrics/clusterRole.yaml

```diff
@@ -0,0 +1,117 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/name: kube-state-metrics
+    app.kubernetes.io/version: 2.0.0-rc.0
+  name: kube-state-metrics
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  - secrets
+  - nodes
+  - pods
+  - services
+  - resourcequotas
+  - replicationcontrollers
+  - limitranges
+  - persistentvolumeclaims
+  - persistentvolumes
+  - namespaces
+  - endpoints
+  verbs:
+  - list
+  - watch
+- apiGroups:
+  - extensions
```

Comment:
Do we really need extensions?

Reply:
Will double-check good catch, we have it in https://github.com/kubernetes/kube-state-metrics/blob/master/examples/standard/cluster-role.yaml#L27-L31